src/transformers/cache_utils.py

```diff
@@ -82,6 +88,140 @@ def seen_tokens(self):
             return None
 
 
+@dataclass
```

Comment:
Missing some copied from here no? Things can be re-used from the other config classes?